webpack/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../tsconfig.json"
```

Comment:
Why to extend base `../tsconfig.json`? Do we use it somewhere else except `webpack`?

Reply:
Because `tsconfig.json` files are only about files at the same level or within folders that are added under `include`. Since I moved the webpack files to their own folder I either had to make another `tsconfig.json` (which is what I did) or add `webpack/` to the top level `tsconfig.json` (which felt wrong because it seems to break the abstraction of which code we care about).